### PR TITLE
Remove the event listeners and replace with event subscriber

### PR DESCRIPTION
**Pull request by dmouse**
* Create new event subscriber classes

### CODE REVIEW

File: src/EventSubscriber/ShowGeneratedFiles.php [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+ */
+
+namespace Drupal\AppConsole\EventSubscriber;
+
+use Drupal\AppConsole\Command\Helper\TranslatorHelper;
+use Symfony\Component\Console\ConsoleEvents;
+use Symfony\Component\Console\Event\ConsoleTerminateEvent;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+class ShowGeneratedFiles implements EventSubscriberInterface
```
Comment (dmouse) [inline src/EventSubscriber/ShowGeneratedFiles.php]: @jmolivas the class name is correct to the event? or any suggestion?

File: src/EventSubscriber/WelcomeMessage.php [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+ */
+
+namespace Drupal\AppConsole\EventSubscriber;
+
+use Drupal\AppConsole\Command\Helper\TranslatorHelper;
+use Symfony\Component\Console\ConsoleEvents;
+use Symfony\Component\Console\Event\ConsoleCommandEvent;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+class WelcomeMessage implements EventSubscriberInterface
```
Comment (dmouse) [inline src/EventSubscriber/WelcomeMessage.php]: @jmolivas the class name is correct for the event? or any suggestion?

Comment (jmolivas) [inline src/EventSubscriber/WelcomeMessage.php]: How about ShowWelcomeMessage instead @dmouse

File: src/EventSubscriber/ShowGeneratedFiles.php [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+        if (method_exists($command, 'getMessages')) {
+            $messages = $command->getMessages();
+
+            foreach ($messages as $message) {
+                $command->showMessage($output, $this->trans($message));
+            }
+        }
+
+        if (method_exists($command, 'getGenerator') && method_exists($command, 'showGeneratedFiles')) {
```
Comment (dmouse) [inline src/EventSubscriber/ShowGeneratedFiles.php]: @jmolivas, Why not create a new class or interface to implement this methods?

File: src/EventSubscriber/ShowGeneratedFiles.php [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+            return;
+        }
+
+        $completedMessageKey = 'application.console.messages.completed';
+
+        if ('self-update' == $command->getName()) {
+            return;
+        }
+
+        if (method_exists($command, 'getMessages')) {
```
Comment (dmouse) [inline src/EventSubscriber/ShowGeneratedFiles.php]: In this comparation I think that is better ask if is an instance of Drupal\AppConsole\Command\Command, what do you think about? @jmolivas

File: src/EventSubscriber/WelcomeMessage.php (hunk 2 above)
Comment (dmouse) [inline src/EventSubscriber/WelcomeMessage.php]: :+1:

**Pull request merged**